apps/ecommerce/providers/shopify/src/app.ts

```diff
@@ -10,7 +12,10 @@ app.use(
   })
 );
 
-app.get('/api', async (req, res) => {
+app.use('/', MetadataController);
+app.use('/resourceTypes', ResourceTypeController);
```

Comment:
nit: according to our technical brief, I believe that this should be `/resourcesTypes`